js/src/components/contact-information/index.js

```diff
@@ -67,15 +67,13 @@ export function ContactInformationPreview() {
 }
 
 /**
- * Renders a contact information section with specified initial state and texts, determined by the `view` prop.
+ * Renders a contact information section with specified initial state and texts.
  *
  * @param {Object} props React props.
- * @param {'setup-mc'|'settings'} props.view Indicate where this component is used.
  * @param {Function} [props.onPhoneNumberVerified] Called when the phone number is verified.
  */
-export default function ContactInformation( { view, onPhoneNumberVerified } ) {
+export default function ContactInformation( { onPhoneNumberVerified } ) {
 	const phone = useGoogleMCPhoneNumber();
-	const isSetupMC = view === 'setup-mc';
```

Comment:
💅 
The related code within the comment needs an update as well.

https://github.com/woocommerce/google-listings-and-ads/blob/4de904a98e44aa45d839fb86043975feba0aba6e/js/src/components/contact-information/index.js#L82-L84

Reply:
Addressed in https://github.com/woocommerce/google-listings-and-ads/pull/1029/commits/920ebd6ec520387996c9685e2ec466ebe7e11c57